content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1006,6 +1006,63 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+### Success policy {#success-policy}
```

Comment:
IMO this should be closer to the `Pod failure policy` section. Maybe we could add it next to `Completion mode` or `Job termination and cleanup`?

Reply:
@atiratree I totally agree with you, but `Pod Failure Policy` exists as a sub-topic for the `Handling Pod and container failures`. So, I put this success policy here.

How about renaming `Handling Pod and container failures` to `Handling Pod and container successes and failures` and moving this section into that section?

Reply:
I think the `Handling Pod and container failures` section is quite packed already. A similar section for success/completion might be better. And possibly combine it with `Completion mode`.

It would be good to also gather an opinion on this from others @alculquicondor @soltysh @kannon92 @mimowo

Reply:
That makes sense. Please let me wait to gather opinions from Job folks.

Reply:
My thoughts:
1. It is Alpha for now so I think "Advanced usage" is the right section.
2. We should not put it under `Handling Pod and container failures`. For one it is packed, second the subject is rather off.
3. I like the idea of "Job success criteria" section, which could wrap both the "Completion mode" and "Success policy".

Reply:
@mimowo It makes sense. In the alpha stage, keep putting successPolicy in the current section.
So, let me add a comment so that we can remember that we should move the successPolicy section to the "Job success criteria" section in the beta stage.

@atiratree WDYT?

Reply:
I am not sure if it makes sense to reorder the documentation later since we would be changing users' UI. But if you prefer, we can do it postpone it...

Reply:
Alpha has nothing to do with the "advanced usage" section. It's more about features that first time users shouldn't need. However, we can argue that the current list of features there don't necessarily satisfy this criteria.

That said, I think we can just put this in its own section right before `Job termination and cleanup`

Reply:
> That said, I think we can just put this in its own section right before Job termination and cleanup

I don't have a strong point of view about places. So, I will put SuccessPolicy section before `Job termination and cleanup`.

Reply:
Moved this section to before `JOb termination and cleanup`.